src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -917,4 +918,59 @@ private void ParseMultipartName()
 
         internal static readonly MultiPartTableName Null = new MultiPartTableName(new string[] { null, null, null, null });
     }
+
+    internal static class SslProtocolsHelper
+    {
+        private static string ToFriendlyName(this SslProtocols protocol)
+        {
+            string name;
+
+            /* The SslProtocols.Tls13 is supported by netcoreapp3.1 and later
+             * This driver does not support this version yet!
+            if ((protocol & SslProtocols.Tls13) == SslProtocols.Tls13)
+            {
+                name = "TLS 1.3";
+            }*/
+            if((protocol & SslProtocols.Tls12) == SslProtocols.Tls12)
```

Comment:
A quick question what if we use switch statement instead of if and else for sslProtocols and inside each case we check the value of protocol? I think Switch is a better option for perf.

Reply:
The `switch` statement is great for fixed data values. If there are only a few cases, it might not affect the speed in any case.